Documentation/sources/content-card-ui/public-classes/uiaction.md

```diff
@@ -0,0 +1,103 @@
+# Sealed Class - UIAction
+
+Represents an action that can be performed on a UI component.
+
+#### Class Parameters
+
+| Parameter | Type                                         | Description                                                  |
+| --------- | -------------------------------------------- | ------------------------------------------------------------ |
+| T         | [AepUITemplate](./UIModels/aepuitemplate.md) | Represents a UI template model associated with the acction like SmallImageTemplate, which backs the composable on which the event has occurred. |
```

Comment:
nit: typo `acction` -> `action`

Reply:
nit: Don't think this part `associated with the acction`  makes sense here, can we remove it? `Represents a UI template model which backs the composable on which the event has occurred.`